source/Calamari.AzureResourceGroup/DeployAzureResourceGroupBehaviour.cs

```diff
@@ -128,14 +123,19 @@ async Task<ArmOperation<ArmDeploymentResource>> CreateDeployment(ArmClient armCl
             }
         }
 
-        async Task PollForCompletion(ArmOperation<ArmDeploymentResource> deploymentOperation)
+        async Task PollForCompletion(ArmOperation<ArmDeploymentResource> deploymentOperation, IVariables variables)
         {
+            var pollingTimeoutVariableValue = variables.Get(SpecialVariables.Action.Azure.ArmDeploymentTimeout, "30");
+            int.TryParse(pollingTimeoutVariableValue, out var pollingTimeoutValue);
```

Comment:
what if I configure my variable with a non-integer value? do we need an `if` here otherwise `pollingTimeoutValue` will be `null`?